src/site/filters/liquid.js

```diff
@@ -68,4 +68,68 @@ module.exports = function registerFilters() {
     const id = `vha_${facilityLocatorApiId}`;
     return JSON.stringify(id);
   };
+
+  // Find the current path in an array of nested link arrays and then return it's depth + it's parent and children
+  liquid.filters.findCurrentPathDepth = (linksArray, currentPath) => {
+    const getDeepLinks = (path, linkArr) => {
```

Comment:
Did you have any luck with Aurora's recursive approach she posted in Slack? I think that's the right approach for something like this, but I understand if we're crunched for time.

Reply:
I gotta re-work it a bit to get the data structure I need but I think this needs to be merged in ASAP if it's working. And then I can push up a refactor post-launch if that works.